src/libratbag-private.h

```diff
@@ -496,6 +498,13 @@ ratbag_resolution_set_resolution(struct ratbag_resolution *res,
 	res->dpi_y = dpi_y;
 }
 
+static inline void
+ratbag_resolution_set_disabled_status(struct ratbag_resolution *res,
+                                      bool disabled)
+{
+	res->is_disabled = disabled;
```

Comment:
I this function even necessary? Or can I also just execute this in `driver-hidpp20.c`?

Reply:
This function is not needed.